src/views/Not-Found.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
Consider renaming this file respecting JavaScript standards (camel care or Pascal case) => NotFound.js